src/Commands/EnvMergeCommand.php

```diff
@@ -65,10 +65,11 @@ public function mergeBuildEnv($site_env_id, $options = ['label' => '', 'delete'
         $tmpMergeBranch = 'temp-local-copy-of-pantheon-master';
 
         // Replace the entire contents of the master branch with the branch we just tested.
+        // @todo, is a merge even necessary when the master branch will be force pushed over?
         $this->passthru('git fetch pantheon');
-        $this->passthru('git checkout pantheon/master');
+        $this->passthru('git checkout pantheon/' . $env_label);
         $this->passthru("git checkout -B $tmpMergeBranch");
-        $this->passthru("git merge -q -m 'Merge build assets from test $env_label.' -X theirs $env_id");
+        $this->passthru("git merge -q -m 'Merge build assets from test $env_label.' --strategy=ours pantheon/master");
```

Comment:
@stevector are we expecting changes in `pantheon/master` that are not in the pull request branch? I guess I'm not clear why we are overwriting the `master` branch.

`git merge --strategy=ours` says
> This resolves any number of heads, but the resulting tree of the merge is always that of the current branch head, effectively ignoring all changes from all other branches. It is meant to be used to supersede old development history of side branches. Note that this is different from the -Xours option to the recursive merge strategy.

`master` shouldn't have old development history. If we are completely replacing `master` with the pull request branch then it would be easier to do
```
git checkout master
git reset --hard pantheon/$env_label
git push --force -q pantheon $tmpMergeBranch:master
```